dags/process_elife_zip_dag.py

```diff
@@ -19,17 +19,15 @@
 from lxml.builder import ElementMaker
 from PIL import Image
 
-from aws import get_aws_connection
-from task_helpers import get_previous_task_name
-
-ALLOWED_EXTENSIONS = {'.jpg', '.jpeg', '.pdf', '.xml'}  # only upload these file types
-IMAGE_EXTENSIONS = {'.jpg', '.jpeg', '.png', '.tif', '.tiff'}
+from aws import get_aws_connection, list_bucket_keys_iter
+from task_helpers import get_file_name_passed_to_dag_run_conf_file
 
 BASE_URL = configuration.conf.get('libero', 'base_url')
 SOURCE_BUCKET = configuration.conf.get('libero', 'source_bucket')
 DESTINATION_BUCKET = configuration.conf.get('libero', 'destination_bucket')
 SERVICE_NAME = configuration.conf.get('libero', 'service_name')
 SERVICE_URL = configuration.conf.get('libero', 'service_url')
+TEMP_DIRECTORY = configuration.conf.get('libero', 'temp_directory') or None
```

Comment:
`None` means the Python standard library will choose where, I guess

Reply:
to quote the python documentation:
```If dir is not None, the file will be created in that directory; otherwise, a default directory is used. The default directory is chosen from a platform-dependent list, but the user of the application can control the directory location by setting the TMPDIR, TEMP or TMP environment variables.```